library/src/meta/figure.rs

```diff
@@ -70,9 +70,41 @@ use crate::visualize::ImageElem;
 /// )
 /// ```
 ///
+/// If your figure is too large and its contents are breakable across pages
+/// (e.g. if there is a table inside the figure, whose rows can be split across
+/// pages), then you can make your figure breakable across pages as well by
+/// adding `#show figure: set block(breakable: true)` before your figure. Note
+/// that such a rule will affect all figures after it, so you may restrict it
+/// to only one figure by wrapping both the show rule and the figure inside a
+/// content block, like below. Note that, when a figure is split across pages,
+/// its caption currently only appears at the last page it spans.
+///
+/// ```example
+/// #[
+///     // By wrapping both the show rule and the
+///     // figure inside a single content block,
+///     // denoted by #[ ], we ensure this show
+///     // rule will only be applied to the
+///     // following figure, so that only that one
+///     // will become breakable with this rule!
+///     #show figure: set block(breakable: true)
+///
+///     #figure(
+///         table(
+///             rows: (auto, auto, 1.5em),
+///             [My very long table:],
+///             v(100% - 0.5em),
+///             [This should appear on the next page]
+///         ),
+///         caption: "Split table!"
+///     )
+/// ]
+///
+/// ```
+///
```

Comment:
Isn't this a bit much? I think just a few sentences plus the relevant show rule would suffice. In particular, I see that you want to show how to do it for just one figure, but these kinds of blocks are so unpretty :/ (and the indent is wrong :p)

In general, I would say that more documentation is good if it covers more things, but individual things should be explained as concisely as possible.

Reply:
That's a great point, thanks. I'll get on it ASAP.

(Sorry for the indenting, my IDE isn't helping me much here hahaha)

Reply:
Done! Made it shorter. Feel free to adjust it as you like.